src/elements/ngm-side-bar.ts

```diff
@@ -406,6 +407,13 @@ export class SideBar extends LitElementI18n {
       layer.visible = true;
       layer.displayed = true;
       this.activeLayers.push(layer);
+      // display visibility hint for subsurface layers
+      if (this.displayUndergroundHint
+        && [LayerType.tiles3d, LayerType.earthquakes].includes(layer.type!)
+        && !this.viewer?.scene.cameraUnderground) {
+        showSnackbarInfo(i18next.t('lyr_subsurface_hint'));
```

Comment:
according to the description, display time should be the same as in the navigation hint (20 sec)

Reply:
addressed in https://github.com/swissgeol/ngm/pull/786/commits/db524154538145c673aa40074b9879a8f272fedd